src/container/sidebar.jsx

```diff
@@ -456,8 +456,14 @@ export const Sidebar = () => {
             return named[0].position.name
         }
         else if(mappings.length == 0) {
-            return mappings[0].position.name
-        } else {
+            if(mappings[0] == undefined) {
```

Comment:
better to do `mappings.length == 0`

Reply:
Done, do I need the else {} code below line 459?

```jsx
454        const named = mappings.filter(mapping => !!mapping.position.name);
455        if(named.length > 0) {
456            return named[0].position.name
457        }
458        else if(mappings.length == 0) {
459            if(mappings.length == 0) {
460                return "Bruker";
461            } else {
462                return mappings[0].position.name
463            }  
464        } 
465        
466        else {
467            return `${mappings.length} stillinger`
468        }
```